tests/functional/spec/difference.spec.js

```diff
@@ -0,0 +1,73 @@
+import AgeBlockYearPage from "../generated_pages/difference_in_years/age-block.page";
+import AgeBlockMonthYearPage from "../generated_pages/difference_in_years_month_year/age-block.page";
+import AgeBlockDayMonthYearRangePage from "../generated_pages/difference_in_years_range/date-block.page";
+import AgeBlockMonthYearRangePage from "../generated_pages/difference_in_years_month_year_range/date-block.page";
+
+describe("Difference check (years)", () => {
+  before("Load the survey", () => {
+    browser.openQuestionnaire("test_difference_in_years.json");
+  });
+
+  it("Given a day, month and year answer is provided for a date question then the age in years should be calculated and displayed on the page ", () => {
+    $(AgeBlockYearPage.day()).setValue(1);
+    $(AgeBlockYearPage.month()).setValue(1);
+    $(AgeBlockYearPage.year()).setValue(1990);
+    $(AgeBlockYearPage.submit()).click();
+
+    const expectedPageTitle = browser.getTitle();
+    expect(expectedPageTitle).to.equal("You are … old. Is this correct? - Difference between two dates");
```

Comment:
Any reason we are asserting the title?. We should assert the question title with the actual placeholder content.

Reply:
The reason is we would have to change the assertion next year?

Reply:
We should work out the age in JS and assert that, otherwise the test is redundant as it does check the placeholder pipes in correct number.

Reply:
I've added a function using Date API that generates the number of years, hope that solves this problem?

Reply:
Should be okay given we are only working with years difference. Worst case it fails once a year, so no biggie.